docs/SequencesAndCollections.rst

```diff
@@ -55,11 +55,11 @@ represented by the `SequenceType` protocol::
 
   __ http://swiftdoc.org/v3.0/type/AnyIterator/
 
-As you can see, sequence does nothing more than deliver an iterator.
+As you can see, sequence does nothing more than delivering an iterator.
```

Comment:
Is this really a typo? I have a strong feeling that "does" warrants an infinitive.

Reply:
While we are here:
```suggestion
As you can see, a sequence does nothing more than delivering an iterator.
```

Reply:
@AnthonyLatsis Thank you for your comments. And I apologize for using inapporiate term "typo" here.
 
I agree that "does" warrants an infinitive but, I thought that the word "deliver" was being emphasized here. And to do so, I thought "delivering" suits better than "deliver" for that purpose.

May I ask for your further opinion on this?

Reply:
> I agree that "does" warrants an infinitive. I thought that the word "deliver" was being emphasized here. For that reason, I believed "delivering" suits better than "deliver".

I hear you, but we have to be careful not to sacrifice grammar for preference. I am not entirely sure the gerund is inappropriate though; it could be that both fit.

cc @CodaFi

Reply:
@AnthonyLatsis Thank you for your opinion. I reverted accordingly. :)